src/main/java/command/Command.java

```diff
@@ -0,0 +1,23 @@
+package command;
+
+import executor.CommandExecutor;
+
+public class Command {
+
+    String input;
```

Comment:
Any reason why you use the default modifier instead of private as you have a getter method?